mail/console/stubs.go

```diff
@@ -0,0 +1,31 @@
+package console
+
+type Stubs struct {
+}
+
+func (s Stubs) Mail() string {
```

Comment:
Missing the `Queue` method?

Reply:
Queue method is only required when you need to queue the mail. So it's optional.